Sources/Clickstream/Configuration/AWSClickstreamPluginConfiguration.swift

```diff
@@ -14,15 +14,11 @@ struct AWSClickstreamConfiguration {
     static let isTrackAppExceptionKey = "isTrackAppExceptionEvents"
     static let isCompressEventsKey = "isCompressEvents"
 
-    static let defaultSendEventsInterval = 10_000
-    static let defaulTrackAppException = true
-    static let defaulCompressEvents = true
-
     let appId: String
     let endpoint: String
     let sendEventsInterval: Int
-    let isTrackAppExceptionEvents: Bool
-    let isCompressEvents: Bool
+    let isTrackAppExceptionEvents: Bool!
+    let isCompressEvents: Bool!
```

Comment:
Does `Bool!` mean it's not empty?

Reply:
`Bool!` which can be nil in Swift, and it will recognized not empty for compiler, we use this [method](https://github.com/awslabs/clickstream-swift/pull/59/files#diff-8bcc7d24673f5e90f01725e28d47bdf4c27f773488bdc10266867ff6215f5295R61-R71) to get the all default value when SDK initialization, so these attributes will not be nil in runtime.